app/controllers/alchemy/admin/user_sessions_controller.rb

```diff
@@ -12,6 +17,7 @@ class UserSessionsController < ::Devise::SessionsController
       before_action :check_user_count, :only => :new
 
       helper 'Alchemy::Admin::Base'
+      helper 'Alchemy::Admin::Form'
```

Comment:
Why is this necessary?

Reply:
I just tested it in the repo and all those includes were necessary for one reason or another.
The form helpers for rendering the re-login views with `alchemy_form_for`

Where does `Devise::SessionsController` inherit from  `Alchemy::BaseController` ?